syncer/sharding-meta/shardmeta.go

```diff
@@ -0,0 +1,232 @@
+// Copyright 2019 PingCAP, Inc.
```

Comment:
maybe we can move more files about sharding DDL into this folder (even rename `sharding-meta` to a better name) later.

This comment just records what I've thought, you can *Resolve* it after you read it.